File: 프로그래머스/1/12934. 정수 제곱근 판별/정수 제곱근 판별.java
class Solution {
    public long solution(long n) {
        long answer = 0;
        long root = (long)Math.round((Math.sqrt(n)));

        if(Math.pow(root,2)==n)
            answer = (long)Math.pow(root+1,2);
        else
            answer = -1;
            
        return answer;
    }
}

File: 프로그래머스/1/12934. 정수 제곱근 판별/정수 제곱근 판별.js
function solution(n) {
    var answer = 0;
    
    console.log(Math.sqrt(n));
    
    if(Math.sqrt(n) == Math.floor(Math.sqrt(n))){
        return (Math.sqrt(n)+1)**2;
    }else{
        return -1;
    }

}